model/riscv_device_tree.sail

```diff
@@ -81,107 +81,98 @@ function ext_wrap(ext : extension, fmt : ISA_Format) -> string = {
 }
 
 function generate_isa_string(fmt : ISA_Format) -> string = {
-  // The Sail compiler blows up if this string is constructed in one go.
-  // Build it up in pieces.
```

Comment:
I think keeping a comment on why we don't use the string concatenation operator here might be useful for future readers. But feel free to ignore this nit.

Reply:
I do plan to fix it, so hopefully that comment would become outdated soon... At least as best as I can, I suspect if one is sufficiently determined it will always be possible to define something so ambiguous the compiler blows up, and that's just the price we pay for allowing fairly arbitrary ad-hoc overloading.